ratings/src/main/java/com/decathlon/vitamin/ratings/VitaminRatingView.kt

```diff
@@ -0,0 +1,216 @@
+package com.decathlon.vitamin.ratings
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.res.ColorStateList
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.LinearLayout
+import androidx.annotation.DrawableRes
+import androidx.annotation.StyleRes
+import androidx.appcompat.content.res.AppCompatResources
+import androidx.core.widget.TextViewCompat
+import com.decathlon.vitamin.ratings.databinding.ViewRatingBinding
+
+
+class VitaminRatingView @JvmOverloads constructor(
```

Comment:
Add `open` to allow extending the class for consumers.

Reply:
Changed